modules/imgcodecs/src/loadsave.cpp

```diff
@@ -60,18 +61,16 @@ namespace cv {
 
 // TODO Add runtime configuration
 #define CV_IO_MAX_IMAGE_PARAMS (50)
-#define CV_IO_MAX_IMAGE_WIDTH (1<<20)
-#define CV_IO_MAX_IMAGE_HEIGHT (1<<20)
```

Comment:
Please remove "TODO" comment too and perform reading of these options once ([example](https://github.com/opencv/opencv/blob/3.4.4/modules/core/src/ocl.cpp#L223-L226), but probably inside of `validateInputImageSize()` function).